addons/account/models/chart_template.py

```diff
@@ -256,8 +256,6 @@ def _load(self, sale_tax_rate, purchase_tax_rate, company):
             'default_cash_difference_expense_account_id': acc_template_ref.get(self.default_cash_difference_expense_account_id.id, False),
             'account_journal_suspense_account_id': acc_template_ref.get(self.account_journal_suspense_account_id.id),
             'account_cash_basis_base_account_id': acc_template_ref.get(self.property_cash_basis_base_account_id.id),
-            'income_currency_exchange_account_id': acc_template_ref.get(self.income_currency_exchange_account_id.id),
-            'expense_currency_exchange_account_id': acc_template_ref.get(self.expense_currency_exchange_account_id.id),
```

Comment:
We could do like the pos receivable account in the code below, but maybe generalize it for all the elements?